src/main/kotlin/com/neva/gradle/fork/config/Config.kt

```diff
@@ -136,7 +143,7 @@ abstract class Config(val project: Project, val name: String) {
   }
 
   private fun promptValidate() {
-    val invalidProps = prompts.values.filter { !it.valid }.map { it.name }
+    val invalidProps = properties.filter(Property::isInvalid).map { it.name }
     if (invalidProps.isNotEmpty()) {
       throw ForkException("Fork cannot be performed, because of missing properties: $invalidProps."
```

Comment:
`missing or invalid`

I mean, we need to update message because now the cause of exception here is not filled property or property filled but not passing validation, build user should be informed about that`